libraries/standard/json/cbmc/sources/skipGeneric.c

```diff
@@ -0,0 +1,33 @@
+#include "skipGeneric.h"
```

Comment:
There is only the one function which is used by the replacements from the proofs, but is not itself a direct replacement.  Since it is a utility function, I put it here.

Reply:
OK